pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "pybamm-tea"
+authors = [{name = "Julian Evers", email = "julian.evers2@gmail.com"}]
+readme = "README.md"
+dynamic = ["version", "description"]
```

Comment:
you'll need to define `__version__` in the `__init__.py` inside the `pybamm_tea` folder